src/cityreader/cityreader.py

```diff
@@ -20,7 +27,11 @@ def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
+    with open("cities.csv") as csvfile:
+      next(csvfile)
+      reader = csv.reader(csvfile)
+      for row in reader:
+        cities.append(City(str(row[0]), float(row[3]), float(row[4])))
```

Comment:
Heheheheh, the whole float thing was tricky, right? Or maybe not. This part ate one hour of my sprint challenge time. Along with the whole naming conventions being `city` and not `name`, but I see you bypassed that by taking the array index. Well done!